src/cheri_insts.sail

```diff
@@ -474,6 +547,20 @@ function clause execute (CSetBoundsImmediate(cd, cs1, uimm)) = {
 }
 
 union clause ast = CSetBoundsExact : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is set to capability register *cs1* with its
+ * **base** field replaced with *cs1*.**address** and its **length** field
+ * replaced with integer register *rs2*.
```

Comment:
Maybe add something like this?

Unlike [csetbounds] this instruction will trap when the result cannot be represented exactly.

I know it's in the exceptions bit but I think most people will not bother reading that.